frontend/src/components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {StyledDropdown, DropdownItem, Image} from "./Dropdown.styled";
+import { FaBell, FaFlag, FaEyeSlash } from "react-icons/fa";
+
+const Dropdown = () => {
+  const dropdownItems = [
+    {
+      icon: <FaBell />,
+      text: 'Notify me'
+    },
+    {
+      icon: <FaFlag />,
+      text: 'Report this post'
+    },
+    {
+      icon: <FaEyeSlash />,
+      text: "Don't show me this post"
+    }
+  ]
+  const displayDropdownItems = dropdownItems.map((dropdownItem, index) =>
+    <DropdownItem key={index}>
+      <Image>
+        {dropdownItem.icon}
+      </Image>
+      <span>
+        {dropdownItem.text}
+      </span>
+    </DropdownItem>
+  )
```

Comment:
1. Chyba nie masz włączonego prettiera albo husky xD Zrób pull main i npm install w katalogu głównym (nie tym frontend).
2. Nie powinno się robić wartości index jako key. Proponuję wartość textu.
3. Czy span jest potrzebny? 
```suggestion
  const displayDropdownItems = dropdownItems.map((dropdownItem) =>
    <DropdownItem key={dropdownItem.text}>
      <Image>
        {dropdownItem.icon}
      </Image>
        {dropdownItem.text}
    </DropdownItem>
  )
```